app/src/theme/index.js

```diff
@@ -129,7 +129,7 @@ const theme = {
     xxxl: '1366px',
   },
   borders: {
-    borderColor: '#ECEFF1',
+    borderColor: '#DCDFF2',
```

Comment:
See last comment

Reply:
@pimato , what do you think?

Reply:
#DCDFF2 is the new "clickable" border color for Omen, so lets change that for every outline button which is active/clickable!